### PR TITLE
Update README with TL instructions for tagging CI image

### PR DESCRIPTION
**Pull request by dustman9000**
Update README with TL instructions for tagging CI image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -375,12 +375,28 @@ principles when deciding what `{X}.{Y}.{Z}` should be. See https://github.com/op
...
+    # Make code changes and create tag
     git tag image-v1.2.3
-    git push origin --tags
-    # Create PR here
-    # Typically only team leads can push tags to upstream, so they will need to continue by
-    # checking out your fork and then running
-    git push upstream --tags
+
+    # Push tag to origin
+    git push origin image-v1.2.3
```
Comment (2uasimojo) [inline README.md:382]: Does this need `--tags`?

File: README.md [hunk 2]
```diff
@@ -375,12 +375,28 @@ principles when deciding what `{X}.{Y}.{Z}` should be. See https://github.com/op
...
+    git fetch --all --prune
+
+    # create branch from user's fork and branch
+    git checkout -b someuser-branch-name someuser/branch-name
+
+    # create tag
+    git tag -f image-v1.2.3
+
+    # push tag to upstream
+    git push upstream -f image-v1.2.3
```
Comment (2uasimojo) [inline README.md:399]: ditto?

**Review by mjlshen — APPROVED** [review]
/lgtm

File: README.md (hunk 2 above)
Comment (dustman9000) [reply README.md:399]: Nope, we only need to push the tag we just added.

File: README.md (hunk 1 above)
Comment (dustman9000) [reply README.md:382]: No, `--tags` is not needed if you specify the tag name.

Comment (2uasimojo) [reply README.md:382]: Cool, sorry for the noise.

**Pull request merged**